lib/retirement_swap/processor.rb

```diff
@@ -0,0 +1,30 @@
+module RetirementSwap
+  class Processor
+    class NullAlgorithm
+      def process(*args)
```

Comment:
Unused method argument - `args`. If it's necessary, use `_` or `_args` as an argument name to indicate that it won't be used. You can also write as `process(*)` if you want the method to accept any arguments but don't care about them.